hack/dev/userdata/.master-1.tpl.yaml

```diff
@@ -2,14 +2,19 @@ services:
   init:
     cni: flannel
   kubeadm:
+    certificateKey: verysimplesecretyoucanguess.....
```

Comment:
with #489 this can be changed to 'ChangeMe' as there will be no 32 bytes requrement